_posts/2020-01-08-cockpit-210.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Cockpit 210 and Cockpit-podman 12
+author: kkoukiou
+date: 2020-01-08
+tags: cockpit linux
+slug: cockpit-210
+category: release
+summary: Cockpit with Overview, Dashboard and Machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes
+from version 210.
+
+### Overview: Add CPU utilization to usage card
+
+Display CPU usage information in the usage card of the Overview page. For detailed CPU usage information, please
+consult the graphs page.
+
+![Overview](/images/overview-cpu-usage.png)
+
+### Dashboard: Allow adding locked identity to keyring when adding new machines
```

Comment:
```suggestion
### Dashboard: Support SSH identity unlocking when adding new machines
```

Reply:
This might be an improvement, but still feels a bit wrong. Anyone have any other suggestion?